force-app/main/default/translations/pt_BR.translation-meta.xml

```diff
@@ -1164,7 +1164,7 @@
     </customLabels>
     <customLabels>
         <name>RD2_EntryFormInsertSuccessMessage</name>
-        <label>A Doação recorrente "{0}” foi criada.</label>
+        <label>A Doação recorrente &quot;{0}” foi criada.</label>
```

Comment:
This is *probably* fine, but the second quote is a "rich text" version and could be replaced by an HTML entity.

Reply:
This _is_ an entity in the English version.  It should be changed.